refs/types.proto

```diff
@@ -77,6 +77,18 @@ message OwnerID {
   bytes value = 1 [json_name = "value"];
 }
 
+// NeoFS subnetwork identifier.
+//
+// `SubnetID` is a 4-byte long.
+//
+// String representation is base-10 integer value.
+//
+// JSON representation is a JSON number.
```

Comment:
Suffice it to say that it corresponds to the standard Protocol Buffers JSON format. If you want to explicitly describe the format of this message, then this is an object with a numeric field `value`.